dotCMS/src/main/java/com/dotcms/security/apps/AppSecrets.java

```diff
@@ -30,6 +30,13 @@ public String getKey() {
         return secrets;
     }
 
+    public void destroy() {
+        for (Secret secret : secrets.values()) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'secret' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5329810)